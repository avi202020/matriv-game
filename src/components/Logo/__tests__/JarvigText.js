import matrivText from "components/Logo/matrivText";
import React from "react";
import { renderWithTheme } from "utils/testHelpers";

describe("Logo matriv Text", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(<matrivText />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("adds the size and spacing modifiers", () => {
    const tree = renderWithTheme(<matrivText sm spaceLeft />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
