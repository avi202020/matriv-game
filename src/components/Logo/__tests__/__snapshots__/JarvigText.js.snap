// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo matriv Text adds the size and spacing modifiers 1`] = `
<h1
  className="sc-bdVaJa kadOOX"
/>
`;

exports[`Logo matriv Text renders correctly 1`] = `
<h1
  className="sc-bdVaJa duNkzU"
/>
`;
